tests/unit/amo/components/TestSearchForm.js

```diff
@@ -173,7 +173,6 @@ describe(__filename, () => {
 
     const link = root.find('link');
     expect(link).toHaveProp('href', `/${lang}/${clientApp}/opensearch.xml`);
-    expect(link).toHaveProp('title', `Firefox Add-ons for Android (${lang})`);
```

Comment:
We should update the test cases (with new strings) instead of deleting assertions.